src/test/java/core/basesyntax/MyLinkedListTest.java

```diff
@@ -60,11 +60,11 @@ public void testAdd() {
     @Test
     public void testAddManyData() {
         MyLinkedListInterface<String> linkedList = new MyLinkedList<>();
-        for (int i = 0; i < 1000; i++) {
+        for (int i = 0; i < 1001; i++) {
```

Comment:
Tests shouldn't be changed. Revert changes

Reply:
how to revert changes in one file?

Reply:
Please follow this guide https://stackoverflow.com/a/2734035/12734524